.gitignore

```diff
@@ -220,4 +220,6 @@ src/next-gen
 package/nuget/tools
 package/chocolatey/*.nupkg
 lib/dlls/**/*
-*.tgz
\ No newline at end of file
+*.tgz
+
+package-lock.json
```

Comment:
Is this really required? I thought package-lock.json is required so any developer will get the same version using npm ci. Why are we removing it from git?